packages/core/nuxt-theme-module/theme/pages/MyAccount/OrderHistory.vue

```diff
@@ -137,7 +137,7 @@ export default {
 
     return {
       tableHeaders,
-      orders: computed(() => orders ? orders.value.results : []),
+      orders,
```

Comment:
you can get rid of `orders` as you are using `totalOrders` inside template

Reply:
`orders` is used in the `v-for` loop in line 61.